src/aria/widgets/form/list/ListController.js

```diff
@@ -499,8 +499,10 @@ module.exports = Aria.classDefinition({
         calcMoveFocus : function (flowOrientation, focusIndex, numberOfRows, numberOfColumns, keyCode, numberItems) {
 
             var moveFocus = focusIndex;
+            var cfg = this._data.cfg;
+            var waiAria = cfg && cfg.waiAria;
```

Comment:
I would rather read the `waiAria` parameter directly from `this._data.waiAria`
The `cfg` parameter is supposed to contain the configuration for the `@aria:Div` widget surrounding the list, as defined [here](https://github.com/ariatemplates/ariatemplates/blob/master/src/aria/widgets/form/list/CfgBeans.js#L72):
```js
"cfg" : {
   $type : "json:ObjectRef",
   $description : "Configuration which should be used for the div surrounding the widget."
},
```